docs/rendering_system.rst

```diff
@@ -0,0 +1,241 @@
+..
+   Author: Adam Turner
+
+   We can't use :pep:`X` references in this document, as they are relative from
+   the content root.
+   We use :doc:`PEP X <../pep-xxxx>` instead.
+
+
+An Overview of the PEP Rendering System
+=======================================
+
+This document provides an overview of the PEP rendering system, as a companion
+to :doc:`PEP 676 <../pep-0676>`.
+
+
+1. Configuration
+----------------
+
+Configuration is held in three files:
+
+- ``conf.py`` contains the majority of the configuration
+- ``contents.rst`` contains the table of contents directives for Sphinx
```

Comment:
```suggestion
- ``contents.rst`` lists the table of contents directives for Sphinx
```

Avoid repetition (assuming `lists` is appropriate; we can find another word if not